src/test/java/com/example/universe/simulator/entityservice/unit/controllers/PlanetControllerTest.java

```diff
@@ -0,0 +1,311 @@
+        Planet planet = Planet.builder().name("name").build();
+        PlanetDto planetDto = modelMapper.map(planet, PlanetDto.class);
+        given(service.get(id)).willReturn(planet);
+        // act
+        RequestBuilder requestBuilder = MockMvcRequestBuilders
+                .get("/planet/get/{id}", id)
+                .accept(MediaType.APPLICATION_JSON);
+        MockHttpServletResponse response = mockMvc
+                .perform(requestBuilder)
+                .andReturn()
+                .getResponse();
+        // assert
+        verifySuccessfulResponse(response, planetDto);
+        then(service).should().get(id);
+    }
+
+    //@Test
+    //void testGet_entityNotFound() throws Exception {
+    //    // arrange
+    //    UUID id = UUID.randomUUID();
+    //    given(service.get(id)).willThrow(new AppException(ErrorCodeType.ENTITY_NOT_FOUND));
+    //    // act
+    //    RequestBuilder requestBuilder = MockMvcRequestBuilders
+    //            .get("/planet/get/{id}", id)
+    //            .accept(MediaType.APPLICATION_JSON);
+    //    MockHttpServletResponse response = mockMvc
+    //            .perform(requestBuilder)
+    //            .andReturn()
+    //            .getResponse();
+    //    // assert
+    //    verifyErrorResponse(response.getContentAsString(), ErrorCodeType.ENTITY_NOT_FOUND);
+    //    then(service).should().get(id);
+    //}
+
+    @Test
+    void testGetList() throws Exception {
+        // arrange
+        List<Planet> planets = List.of(Planet.builder().name("name").build());
+        List<PlanetDto> planetDtos = modelMapper.map(planets, new TypeToken<List<PlanetDto>>() {
+        }.getType());
+        given(service.getList()).willReturn(planets);
+        // act
+        RequestBuilder requestBuilder = MockMvcRequestBuilders
+                .get("/planet/get-list")
+                .accept(MediaType.APPLICATION_JSON);
+        MockHttpServletResponse response = mockMvc
+                .perform(requestBuilder)
+                .andReturn()
+                .getResponse();
+        // assert
+        verifySuccessfulResponse(response, planetDtos);
+        then(service).should().getList();
+    }
+
+    @Test
+    void testAdd() throws Exception {
+        // arrange
+        Planet planet = Planet.builder().name("name").build();
+        PlanetDto planetDto = modelMapper.map(planet, PlanetDto.class);
+        given(service.add(planet)).willReturn(planet);
+        // act
+        RequestBuilder requestBuilder = MockMvcRequestBuilders
+                .post("/planet/add")
+                .accept(MediaType.APPLICATION_JSON)
+                .contentType(MediaType.APPLICATION_JSON)
+                .content(objectMapper.writeValueAsString(planetDto));
+        MockHttpServletResponse response = mockMvc
+                .perform(requestBuilder)
+                .andReturn()
+                .getResponse();
+        // assert
+        verifySuccessfulResponse(response, planetDto);
```

Comment:
Here you should use `modelMapper.map(entity, PlanetDto.class)`  instead of `planetDto`. Input and output dtos might be different.